### DISCUSSION TITLE
MobilPay and Apple Pay Stripe support

### DISCUSSION THREAD

**Discussion by informif** [medusajs/medusa#7399] [Feature Requests]
We would love to see native support for MobilPay and Apple Pay through Stripe in MedusaJS 2.0. This addition would greatly enhance the payment flexibility and user experience for our e-commerce platform. Integrating these payment methods will streamline the checkout process and cater to a broader range of customers who prefer using MobilPay and Apple Pay.

Benefits:

Improved customer convenience
Expanded payment options
Enhanced user experience
Thank you

**Comment by stephanboersma** [comment]
MobilePay and ApplePay is supported with Reepay/Billwerk plug-in and gateway :)
